torch/csrc/Module.cpp

```diff
@@ -91,13 +89,13 @@ static bool THPModule_assignStateless(PyObject *self)
 #define INIT_STATELESS(type)                                                   \
   stateless = PyObject_Call((PyObject*)&TH_CONCAT_2(type, TensorStatelessType), arg, NULL); \
   if (!stateless) {                                                            \
-    THPUtils_setError("stateless method initialization error");                \
     return false;                                                              \
   }                                                                            \
   if (PyObject_SetAttrString(TH_CONCAT_3(THP,type,TensorClass), THP_STATELESS_ATTRIBUTE_NAME, stateless) == -1) { \
-    THPUtils_setError("stateless method initialization error (on assignment)");\
+    return false;                                                              \
   }
   PyObject *arg = PyTuple_New(0);
+  if (!arg) return NULL;
```

Comment:
Can you change the macro to use `PyObject_CallFunctionObjArgs` and remove that tuple?